build.gradle.kts

```diff
@@ -27,6 +27,7 @@ repositories {
 
 dependencies {
     implementation("com.github.f4b6a3:ulid-creator:2.0.2")
+    implementation("cool.graph:cuid-java:0.1.1")
```

Comment:
Too bad it's already old and archived :/

But as there do not seem to be another JVM implementation available...

Reply:
I wrote a Kotlin implementation, but it's the same code but in Kotlin + MPP, plus it's hosted on my Github Packages and currently it's private; however I think that even if that lib is old, does its job

Reply:
Good to know, that will do it for now